src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -11,6 +11,7 @@ public class CliSyntax {
     public static final Prefix PREFIX_EMAIL = new Prefix("e/");
     public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
-    public static final Prefix PREFIX_INSURANCE_ID = new Prefix("id/");
-
+    public static final Prefix PREFIX_INSURANCE_ID = new Prefix("iid/");
```

Comment:
will this break the add and delete insurance commands since we are refactoring this to `iid`?

Reply:
No i tried using these and it worked since everything was following the variable rather than the value